src/NumericalAlgorithms/Interpolation/PredictedZeroCrossing.hpp

```diff
@@ -28,7 +52,9 @@ double predicted_zero_crossing_value(const std::vector<double>& x_values,
  * Each linear function is fit by M points, where M is the size of x_values and
  * y_values. The Nth point in the returned DataVector is the value of x for
  * which the linear fit to the points (x[:], y[:][N]) [using python notation]
- * crosses y = 0. The x_values contain M points in the independent variable,
+ * crosses y = 0 (but if we cannot determine within the error bars of the
```

Comment:
I found the documentation of this function to be repetitive (and therefore confusing...).   Can't it be simplified to point to the other function, just stating that it calls it for each index of the DataVectors?

Reply:
Ok, I will try to simplify this documentation.

(The reason the documentation is long is that there was a lot of confusion in #4004 as to the sizes of the arrays and sizes of the DataVectors)